index.html

```diff
@@ -5,296 +5,307 @@
-                                    <h4 class="h4-like news__title">Phasellus Lacina</h4>
-                                    <span class="news__comments">22 Comments</span>
-                                    <p lang="en" class="prh news__about">Lospendisse ac urna. Etiam pellentesque mauris ut lectus. Nunc
-                                        tellus ante, mattis eget, gravida vitae, ultricies atinvio
-                                        <span class="news__more-icon icon-arrow-circle-right"></span>
-                                    </p>
-                                </div>
-                            </div>
-                        </li>
-                    </ul>
-                    <ul class="list life__list">
-                        <li class="life__says-list-item ">
-                            <div class="says">
-                                <div class="says__comment">
-                                    <p lang="en" class="prh says__what">
-                                        “ Sed egestas, ante et vulputate volutpat, eros pede 
-                                        semper est, vitae luctus metus libero eu augue. Morbi purus libero, faucibus adipiscing’
-                                    </p>
-                                    <p class="says__who">- John Doe</p>
-                                </div>
-                                <div class="container circle-icon says__icon">
-                                    <img class="says__icon-img" src="./resources/images/comments.svg">
-                                </div>
-                            </div>
-                        </li>
-                        <li class="life__says-list-item ">
-                            <div class="says">
-                                <div class="says__comment">
-                                    <p lang="en" class="prh says__what">
-                                        “ Ced egestas, ante et vulputate volutpat, eros pede
-                                        semper est, vitae luctus metus libero eu augue. Morbi purus libero, faucibus adipiscing’
-                                    </p>
-                                    <div class="says__who">- Kate Doe</div>
-                                </div>
-                                <div class="container circle-icon says__icon">
-                                    <img class="says__icon-img" src="./resources/images/comments.svg">
-                                </div>
-                            </div>
-                        </li>
-                    </ul>
+            </section>
+            <section class="life">
+                <div class="life__grid-1r">
+                    <div class="life__header">
+                        <h2 class="h1-like life__title">Vestibulum Commodo Felis</h2>
+                        <p class="prh life__prh">Morbi interdum mollis sapien. Sed ac risus. Phasellus lacinia, magna</p>
```

Comment:
From the semantics standpoint, it may even be a span.

Reply:
Okey, in this case I had to set some display prop as block or inline-blocke to make line-height work, right?